pkg/trantor/testing/smr_property_test.go

```diff
@@ -129,6 +135,9 @@ func checkEventTraces(eventLogFiles map[t.NodeID]string, transactionsSubmitted i
 		if cnt.Error() {
 			return es.Errorf("could not read number of delivered batches: %w", cnt.E)
 		}
+		if cnt.V == nil {
+			return es.Errorf("no transactions were delivered")
+		}
```

Comment:
Curious about this, can you elaborate on how the changes on quorum size calculation causes an error here?

Reply:
See [comment above](https://github.com/filecoin-project/mir/pull/459#discussion_r1206432303). It doesn't have anything to do with the quorum size calculation, and it would have been cleaner to fix it in a different PR.

I just initially had a bug in the quorum computation that completely prevented the delivery of any batches, which made me discover the issue with the test.